package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webfonts-loader",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
You shouldn't update the version manually. Put it back to 2.0.3 NPM will handle it ;)